.ado/release.yml

```diff
@@ -23,9 +23,9 @@ schedules:
 variables:
   CARGO_TERM_COLOR: always
   RUST_TOOLCHAIN_VERSION: "1.88"
-  PYTHON_VERSION: "3.11"
+  PYTHON_VERSION: "3.12"
   LLVM_VERSION: 14
-  AUDITWHEEL_TAG: "manylinux_2_35"
+  CACHE_COUNTER: 0 # use this to invalidate the cache
```

Comment:
Can you be more specific here. As in, increment it when you want to invalidate the cache? Why might you want to invalidate the cache?

Reply:
The cache key is based on the platform, but I changed the OS image, not the platform. So I needed to invalidate the cache to make sure that we built LLVM with the new image.